### DISCUSSION TITLE
Selft-hosted instance problems

### DISCUSSION THREAD

**Discussion by pluja** [supabase/supabase#6067] [Questions] [answered]
I am trying to set up a self-hosted intance on my own server. I've edited the .env with all that is needed and the Docker-compose starts successfully.

When I access my instance this is what I see:

![](https://i.imgur.com/U1CvOzD.png)

There is no authentication page and there is only a default Project which I haven't created and I can't create any more projects than this one. Also, the API page is giving me an error:

![](https://i.imgur.com/5YP9YJC.png)

My Nginx config is:

```
server {
  server_name sub.domain.tld;

  location / {
    proxy_pass http://localhost:3000/;
  }

      # REST
    location ~ ^/rest/v1/(.*)$ {
        proxy_set_header Host $host;
        proxy_pass http://localhost:8000;
        proxy_redirect off;
    }

    # AUTH
    location ~ ^/auth/v1/(.*)$ {
        proxy_set_header Host $host;
        proxy_pass http://localhost:8000;
        proxy_redirect off;
    }

    # REALTIME
    location ~ ^/realtime/v1/(.*)$ {
        proxy_redirect off;
        proxy_pass http://localhost:8000;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Host $host;
    }
}
....
```

And it works with HTTPS.

**Comment by GaryAustin1** [comment, marked as answer]
About no login: https://github.com/supabase/supabase/issues/4934

About only single project (that is the limit on selfhosted): https://github.com/supabase/supabase/issues/4440#issuecomment-992108832

These might be related to the API page issue: 
https://github.com/supabase/supabase/pull/5605
https://github.com/supabase/supabase/issues/4550
If not, in discussions and issues in this repository are several hits on 401 and api
